 Disable Zygote preloaded drawables.

With a theme applied, most of these preloaded drawables go unused.  Any
assets the theme has redirected will need to be loaded with each app
process regardless.  Worse, preloads make it impossible to do asset
redirection for constituent parts of a preloaded drawable (for instance,
individual states of a StateListDrawable cannot be redirected).

Some day it might be nice to revisit this and see if there's a way to
reintroduce the drawable cache in a way that can be altered at runtime
without significant complexity or runtime penalty.

diff --git a/core/java/com/android/internal/os/ZygoteInit.java b/core/java/com/android/internal/os/ZygoteInit.java
index 7028fa7..2521270 100644
--- a/core/java/com/android/internal/os/ZygoteInit.java
+++ b/core/java/com/android/internal/os/ZygoteInit.java
@@ -108,7 +108,7 @@
     private static final String PRELOADED_CLASSES = "/system/etc/preloaded-classes";
 
     /** Controls whether we should preload resources during zygote init. */
-    public static final boolean PRELOAD_RESOURCES = true;
+    public static final boolean PRELOAD_RESOURCES = false;
 
     /**
      * Registers a server socket for zygote command connections
@@ -437,6 +437,8 @@ private static void preloadResources() {
                     Log.i(TAG, "...preloaded " + N + " resource in "
                             + (SystemClock.uptimeMillis() - startTime) + "ms.");
                 }
+            } else {
+                Log.i(TAG, "Preload resources disabled, skipped.");
             }
             mResources.finishPreloading();
         } catch (RuntimeException e) {
